### PR TITLE
MacOS fixes

### PR DESCRIPTION
**Pull request by jessicafalk**
#### Description
Some fixes to be able to run the GDK on MacOS.
1. fixing a bug that will fail the build
2. updating Setup.sh and the relevant Build project
3. disabling the deployment manager for MacOS until they are compatible

### CODE REVIEW

File: Setup.sh [hunk 1]
```diff
@@ -67,13 +67,19 @@ markStartOfBlock "Setup variables"
...
 
 markStartOfBlock "Clean folders"
     rm -rf $CORE_SDK_DIR           2>/dev/null
     rm -rf $WORKER_SDK_DIR         2>/dev/null
     rm -rf $BINARIES_DIR           2>/dev/null
-    rm -rf $SCHEMA_STD_COPY_DIR    2>/dev/null
+
+    if [ ! -z "$SPATIAL_DIR" ]; then
+        rm -rf $SCHEMA_STD_COPY_DIR    2>/dev/null
+        rm -rf $SCHEMA_COPY_DIR    2>/dev/null
```
Comment (joshuahuburn) [inline Setup.sh]: nit on the whitespace at the end :D

File: SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp [hunk 2]
```diff
@@ -146,6 +148,7 @@ bool FLocalDeploymentManager::ParseJson(const FString& RawJsonString, TSharedPtr
 // For other processes which do not spawn cmd windows, use ExecProcess instead.
 void FLocalDeploymentManager::ExecuteAndReadOutput(const FString& Executable, const FString& Arguments, const FString& DirectoryToRun, FString& OutResult, int32& ExitCode)
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp]: When does this get called on mac?

Comment (jessicafalk) [reply SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp]: good point, reverted the change

File: SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h [hunk 3]
```diff
@@ -28,7 +28,7 @@ struct FConnectionConfig
         
 #if PLATFORM_IOS || PLATFORM_ANDROID
 		// On a mobile platform, you can only be a client worker, and therefore use the external IP.
-		WorkerType = SpatialConstants::ClientWorkerType;
+		WorkerType = SpatialConstants::DefaultClientWorkerType.ToString();
```
Comment (padhansell) [inline SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h]: There's some discussion around this going on in https://improbableio.atlassian.net/browse/UNR-1763

**Pull request merged**